src/parser/priest/discipline/CombatLogParser.js

```diff
@@ -40,7 +40,7 @@ import Schism from './modules/spells/Schism';
 
 import SinsOfTheMany from './modules/spells/SinsOfTheMany';
 
-import Mindgames from './modules/shadowlands/covenants/Mindgames';
+import MindGames from './modules/shadowlands/covenants/MindGames';
```

Comment:
It's supposed to be Mindgames; file needs to be renamed (class inside and other references were already fixed)